src/app/services/age-verification/age-verification.service.spec.ts

```diff
@@ -33,4 +33,20 @@ describe('AgeVerificationService', () => {
 
     expect(result).toContain('is a minor');
   });
+
+  it('should returns false when age is lowerthen legal age', () => {
```

Comment:
this 2 tests below should come first since you need to order `isLegalAge` before `performAgeValidation`